inst/tests/tests.Rraw

```diff
@@ -11983,6 +11983,36 @@ test(1914.8, identical(dtGet(dt.comp, 1), dt[[1]]))
 test(1914.9, identical(dtGet(dt.comp, 'b'), dt$b))
 # END port of old testthat tests
 
+# min/max work for _ordered_ factors (not unordered)
+DT <- data.table(
+  V1=factor(rep(c("a","b"), 10), levels=c("a", "b")),
+  V2=rep(c("c","d", "e", "f"), 5)
+)
+test(1915.1, DT[ , min(V1)], error = 'not meaningful for factors')
+test(1915.2, DT[ , max(V1)], error = 'not meaningful for factors')
+
+## confirming base functionality works
+DT[ , V1 := as.ordered(V1)]
+test(1915.3, DT[ , min(V1)],
+     structure(1L, .Label = c("a", "b"), class = c("ordered", "factor")))
+test(1914.4, DT[ , max(V1)],
+     structure(2L, .Label = c("a", "b"), class = c("ordered", "factor")))
+
+## make sure GForce is activated
+old_optim = options(datatable.optimize = Inf)
+test(1914.5, DT[ , min(V1), by = V2],
+     data.table(
+       V2 = c("c", "d", "e", "f"),
```

Comment:
In both GForce tests you used distinct values to group by in V2 field, which is somehow an edge case when grouping, in case of min/max then it is simply "as is". Would be a little bit safer to have GForce tests with 3 rows matching same group, so min/max on that is actually tested.

Reply:
@MichaelChirico nice catch/fix. If you can ping me when you've added the test you'd like to, I'd be happy to merge.